src/skins/vector/views/molecules/ServerConfig.js

```diff
@@ -20,37 +20,142 @@ var React = require('react');
 var Modal = require('matrix-react-sdk/lib/Modal');
```

Comment:
again, this component has been de-split so is now non longer a view, but is still in the views directory.